doc/classes/Camera3D.xml

```diff
@@ -92,12 +92,12 @@
 				Returns a 3D position in world space, that is the result of projecting a point on the [Viewport] rectangle by the camera projection. This is useful for casting rays in the form of (origin, normal) for object intersection or picking.
 			</description>
 		</method>
-		<method name="set_cull_mask_bit">
+		<method name="set_cull_mask_value">
 			<return type="void" />
-			<argument index="0" name="layer" type="int" />
-			<argument index="1" name="enable" type="bool" />
+			<argument index="0" name="layer_number" type="int" />
+			<argument index="1" name="value" type="bool" />
 			<description>
-				Enables or disables the given [code]layer[/code] in the [member cull_mask].
+				Sets the specified [code]value[/code] of the [member cull_mask] to [code]true[/code] or [code]false[/code], given a [code]layer_number[/code] between 1 and 20.
```

Comment:
This description doesn't make much sense because `[code]value[/code]` refers to the passed bool, not to the value in `cull_mask` associated with the given layer. So I'd change it to something like:
```suggestion
				Sets the specified value in the [member cull_mask] to [code]value[/code], given a [code]layer_number[/code] between 1 and 20.
```
or:
```suggestion
				Sets the value for the specified layer in [member cull_mask] to [code]value[/code], given a [code]layer_number[/code] between 1 and 20.
```
or:
```suggestion
				Based on [code]value[/code], enables or disables the specified layer in the [member cull_mask], given a [code]layer_number[/code] between 1 and 20.
```
---
The same issue is present in other similar changes (I'm not commenting them one by one though).

Reply:
Makes sense! I've updated the documentation based on your third suggestion, and also updated the `get_` functions to use similar wording.